### PR TITLE
[Feature] New Constants & Units

### PR DESCRIPTION
**Pull request by coco33920**
( a problem in the file, maybe because of the Const => Constante commit )

### CODE REVIEW

**Comment by quentinguidee** [comment]
@Gegenter I added you as a reviewer for the HU translations (as usual 😛)

**Comment by coco33920** [comment]
Just a precision, i added all the name and values for the pka couples in shared.universal assume that pKa is used in others country as well, ( if not we'll just need to translate `Pka` because the others tag are only molecules formulas )
Charlotte

**Comment by coco33920** [comment]
It was the last commits, we have 202 constants and 3 new units

File: apps/toolbox.fr.i18n [hunk 1]
```diff
@@ -423,13 +423,73 @@ AvogadroTag = "Constante d'Avogadro"
...
+PlanckImpedanceTag = "Impédance de Planck"
+TauonMassTag = "Tauon"
+WBosonMassTag = "Boson W"
+ZBosonMassTag = "Boson Z"
+FineStructureTag = "Constante de Structure Fine"
+RydbergConstantTag = "Constante de Rydberg"
+HartreeConstantTag = "Constante de Hartree"
+MagneticFluxQuantumTag = "Quantum de Flux Magnétique"
+ConductanceQuantumTag = "Quantum de Conductance"
+CirculationQuantumTag = "Quantum de Circlulation"
```
Comment (quentinguidee) [inline apps/toolbox.fr.i18n]:
```suggestion
CirculationQuantumTag = "Quantum de Circulation"
```

File: apps/toolbox.fr.i18n [hunk 2]
```diff
@@ -423,13 +423,73 @@ AvogadroTag = "Constante d'Avogadro"
...
+Radiuses = "Rayons"
+Length = "Longueur"
+Distances = "Distances"
+EarthMoonDistanceTag = "Terre - Lune"
+EarthSunDistanceTag = "Terre - Soleil"
+FaradayConstantTag = "Constante de Faraday"
+StefanBoltzmannTag = "Constante de Stefan-Boltzmann"
+WaterTriplePointTag = "Point Triple de l'eau"
+WienTag = "Constante de Wien"
+AtmosphericPressureTag = "Pression Atmosphèrique"
```
Comment (quentinguidee) [inline apps/toolbox.fr.i18n]:
```suggestion
AtmosphericPressureTag = "Pression Atmosphérique"
```

**Comment by coco33920** [comment]
I removed all the molar masses from the toolbox, we gained ~39kio when doing so

**Pull request merged**